src/metabase/driver/common/parameters/values.clj

```diff
@@ -305,7 +306,9 @@
   (try
     (parse-value-for-type (:type tag) (parse-tag tag params))
     (catch Throwable e
-      (throw (ex-info (tru "Error determining value for parameter: {0}" (ex-message e))
+      (throw (ex-info (tru "Error determining value for parameter {0}: {1}"
+                           (pr-str (:name tag))
+                           (ex-message e))
```

Comment:
Unrelated to fix, but I tweaked this Exception to include the name of the parameter it failed to parse a value for, which makes debugging errors in embedding easier